micro-rdk-server/esp32/ota_8mb_partitions.csv

```diff
@@ -1,7 +1,8 @@
 # Name,   Type, SubType, Offset,  Size, Flags
 # Note: if you have increased the bootloader size, make sure to update the offsets to avoid overlap
-nvs,	       data,	nvs,	  0x9000,	0x6000,
-otadata,       data,	ota,	  0xF000,	0x2000,
-phy_init,      data,	phy, 	  0x11000, 	0x1000,
-ota_0,	       0,	ota_0,	  ,		0x377000,
-ota_1,	       0,	ota_1,	  ,		0x377000,
+nvs,         data,  nvs,      0x9000,   0x6000,
+otadata,     data,  ota,      0xF000,   0x2000,
```

Comment:
nit: we should try to be consistent about upper vs lower for hex digits (see `0x7e`) below.